ice/gridproviders/consolegrid_provider.py

```diff
@@ -9,10 +9,17 @@
 
 import sys
 import os
-import urllib
-import urllib2
 
-import grid_image_provider
+# Python 2 and 3 compatibility for urllib.
```

Comment:
Oof. I think this is a sign that I need to convert over to something like `requests`.